content/blog/tinasaurus-docusaurus-starter.mdx

```diff
@@ -7,7 +7,7 @@ author: Scott Gallant & Scott Byrne
 
 We're excited to share [Tinasaurus](https://github.com/tinacms/tinasaurus ""), a Docusaurus starter site with support for TinaCMS.
 
-<WideImage src="https://res.cloudinary.com/forestry-demo/image/upload/v1677522334/tina-io/blog/tinacms-docusaurus-markdown-editing.gif" alt="Editing Docusaurus content with a CMS" />
+<Youtube embedSrc="/img/blog/tinasaurus-docusaurus-starter/tinacms-docusaurus-markdown-editing.webm"/>
```

Comment:
**question**: Is this the correct replacement component, given that the movie does not autoplay?

https://tina.io/blog/tinasaurus-docusaurus-starter/

![image](https://github.com/user-attachments/assets/f90931fb-a228-4172-a056-d8ea238641f3)
**Figure: For the updated site on the left, the movie is poorly proportioned with plenty of black above and below. The movie also does not autoplay.**

Reply:
Fixed - changed to `<WebmEmbed/>`